LWOTC_WeaponPack/Src/LWOTC_WeaponPack/Classes/X2Item_BRSchematics.uc

```diff
@@ -0,0 +1,222 @@
+//---------------------------------------------------------------------------------------

```

Comment:
I know it's a pain, but can you put all of this in the con/las/mag/coil/beam schematics file?